architecture/additional_concepts/contiv.adoc

```diff
@@ -13,22 +13,43 @@ toc::[]
 [[architecture-additional-concepts-contiv-overview]]
 == Overview
 
-*Contiv* is an open-source networking plugin module for container infrastructure. 
-*Contiv* provides an infrastructure for application-oriented network policies and support for a range of multiple networking modes. These include a configurable set of overlay networking modes, physical networking modes and support for industry-leading hardware.  {product-title} can use it for networking containers instead of the default Openshift SDN. Full technical details on *contiv* are available at the 
-link:http://contiv.github.io[Contiv github pages] 
+link:http://contiv.github.io[Contiv] is an open-source networking plug-in module
+for container infrastructure. Contiv provides an infrastructure for
+application-oriented network policies and support for a range of multiple
+networking modes. These include:
 
-[[architecture-additional-concepts-contiv-architecture]]
-== Architecture
-
-Each node within the cluster runs a *contiv* agent called *netplugin* while the master hosts run the *contiv* controller (referred to as *netmaster*) alongwith supporting control plane components (such as *etcd*). 
+- A configurable set of overlay networking modes.
+- Physical networking modes.
+- Support for industry-leading hardware.
 
-Together the components of *contiv (*netmaster* and *netplugin*) handle key networking functions for Openshift including 
+{product-title} can use Contiv for networking containers instead of the default
+xref:../../architecture/additional_concepts/sdn.adoc#architecture-additional-concepts-sdn[OpenShift
+SDN].
 
-- Assigning IP addresses to each container pod on each cluster node 
-- Creating and managing multiple separate container network instances for different groups of containers 
-- Configuring the network forwarding layer components for layer 2 or layer 3 forwarding
-- Configuring and enforcing a range of network policies that provide  
-- Providing management interfaces (including both CLI and GUI) to configure and manage contiv-specific features and configurations
-- Providing an infrastructure for role based controls that allow for multiple role based network operations workflows. 
+[[architecture-additional-concepts-contiv-architecture]]
+== Architecture
 
-*Contiv* uses the Container Network Interface (link:https://kubernetes.io/docs/admin/network-plugins/#cni[CNI]) to interface with Openshift/ Kubernetes. A key value store based on *etcd* is used to store contiv-specific state information. This is in addition to and separate from the instance of *etcd* used by other components in the system including Openshift itself. 
+Each node within the cluster runs a Contiv agent called *netplugin* while the
+master hosts run the Contiv controller (*netmaster*), along with supporting
```

Comment:
Maybe make the (netmaster) into a "called netmaser" style, to better match the "calling netplugin" before it.